Sources/NetworkService/ParametersEncoder/URLParametersEncoder.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+public struct URLParametersEncoder: ParametersEncoder {
```

Comment:
Should you add documentation here?
```suggestion
/// Encodes URL parameters into a request's URL.
public struct URLParametersEncoder: ParametersEncoder {
```

Reply:
Fixed.